Apps/Patient/src/Controllers/PatientController.cs

```diff
@@ -69,9 +75,14 @@ public PatientController(IPatientService patientService, Services.IPatientServic
         [ApiVersion("1.0")]
         [Route("{hdid}")]
         [Authorize(Policy = PatientPolicy.Read)]
-        public async Task<RequestResult<PatientModel>> GetPatient(string hdid)
+        public async Task<RequestResult<PatientModelV1>> GetPatient(string hdid)
```

Comment:
Could we have not left this untouched and changed GetPatientV2 to return PatientModelV2 instead?

Note: eventually once we have problem details exception handling resolved, the ClientRegistriesDelegete and PatientService in Patient were going to be removed and pointed at Common.